packages/git/src/browser/dirty-diff/diff-computer.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as jsdiff from 'diff';
+
+export class DiffComputer {
+
+    computeDiff(previous: string[], current: string[]): DiffResult[] {
+        const diffResult = jsdiff.diffArrays(previous, current);
+        return diffResult;
+    }
+
+}
+
+export class DirtyDiffComputer extends DiffComputer {
+
+    computeDirtyDiff(previous: string[], current: string[]): DirtyDiff {
```

Comment:
couldn't this method just be added to `DiffComputer`

Reply:
well, it could be just `DiffComputer`.

Reply:
done.